### PR TITLE
Added nmap-aggressive features

### PR DESCRIPTION
**Pull request by theabraxas**
also added: chmod +x drupal detection script, and updated README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,7 +130,9 @@ Battalion also supports configuring the locations of tools via environment varia
 - `--email-domain <domain name>`: Allows a different email domain to be configured
 - `--subdomain-list <file>`: Specify a file that provides potential subdomains
 - `--nmap`: Enable light touch nmap scanning of subdomains
+- `--nmap-aggressive`: This is a VERY intense scan on each subdomain, approx ~10 minutes per subdomain. VERY LONG TO RUN.
 - `--shodan <api key>`: Specify a Shodan API key and enables a Shodan scan
+- `--hunter <api key>`: Sets a Hunter.io API Key and enables Hunter in the user scan. This will vastly speed up the user scan!
```
Comment (eidolonpg) [inline README.md]: Why is this showing up as an add? This should already be in master...

Comment (theabraxas) [inline README.md]: You added it to battalion.sh but not the README

File: battalion.sh [hunk 2]
```diff
@@ -94,6 +95,9 @@ do
         --nmap)
             NMAP_ENABLED=true
             ;;
+        --nmap-aggressive)
```
Comment (eidolonpg) [inline battalion.sh]: This is fine for now. Eventually we should consider doing something like this:

```
--nmap [light|medium|aggressive]
```

Or something to that effect if we add more control over our scan.

File: domain-scan/domain-scan.sh [hunk 3]
```diff
@@ -106,9 +106,18 @@ if $NMAP_ENABLED ; then
     echo ""
     echo "> Executing Nmap scan on all subdomains."
 
-    cat $SUBDOMAIN_LIST  | jq -M -r -c '.subdomains | .[]' | uniq -i | $DOMAIN_SCAN_SCRIPTS/nmap-basic.sh $NMAP_DIRECTORY
+    cat $SUBDOMAIN_LIST | jq -M -r -c '.subdomains | .[]' | uniq -i | $DOMAIN_SCAN_SCRIPTS/nmap-basic.sh $NMAP_DIRECTORY
+
+    echo -e "\t- Produced $(ls -1 ${NMAP_DIRECTORY} | wc -l) Nmap reports."
```
Comment (eidolonpg) [inline domain-scan/domain-scan.sh]: Now with the XML being generated this number is technically doubled.

Comment (theabraxas) [inline domain-scan/domain-scan.sh]: Oh good point =/

**Pull request merged**